README.md

```diff
@@ -1,72 +1,57 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome books with ES6.
 
-> Description the project.
+> Hello! I am a software developer! I can help you build a product, feature or website. Tak a look of my works.
+If you like what you see and have a project you need coded, don’t hesitate and contact me.
```

Comment:
- [ ] Please kindly update your project description to reflect the current project, as this will help users to quickly get an idea of what your project is about.